### PR TITLE
Rule dir tools tests

### PR DESCRIPTION
**Pull request by cipherboy**
A few tests against the functions in `ssg/rule_dir_stats.py`. Not against everything, but something to get started with. :)

### CODE REVIEW

File: tests/unit/ssg/test_rule_dir_stats.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    one_rule = {
+        "id": "one_rule",
+        "ovals": {
+            "rhel6.xml": {},
+        }
+    }
+    empty_rule = {
+        "id": "bad_rule",
+        "ovals": {}
+    }
```
Comment (yuumasato) [inline tests/unit/ssg/test_rule_dir_stats.py]: I have a feeling that it makes sense to start testing from `rule_dir_json.py`, and have all these rules in `data` dir.

Comment (cipherboy) [reply tests/unit/ssg/test_rule_dir_stats.py]: Sure, I think this'll require a code restructuring though, as `rule_dir_json` doesn't currently use code in the `ssg` folder... So we're not guaranteeing order of execution of the unit tests for `utils` vs `ssg`... Which means we can't depend on `rule_dir_json.py` executing (and output existing) before these checks unless we call it ourselves. 

Thoughts?

Comment (yuumasato) [reply tests/unit/ssg/test_rule_dir_stats.py]: @cipherboy I'm probably mixing things up, what I had in mind was more like integration tests. To load the rules from `data`dir and test the `stats` script.
As these are unit tests, there test rules are fine like this.

Comment (cipherboy) [reply tests/unit/ssg/test_rule_dir_stats.py]: Ahh yes, integration tests would be good to have, I agree. I'll add them in a separate PR. :)

**Comment by yuumasato** [comment]
Looks good, sorry for delay.

**Pull request merged**